engine/src/cubos/engine/tools/scene_editor/plugin.cpp

```diff
@@ -81,7 +81,7 @@ static void openScene(const Asset<Scene>& sceneToSpawn, Commands& commands, cons
     closeScene(commands, scene);
     auto sceneRead = assets.read(sceneToSpawn);
     auto builder = commands.spawn(sceneRead->blueprint);
-    for (const auto& [entity, name] : sceneRead->blueprint.getMap())
+    for (const auto& [entity, name] : sceneRead->blueprint.bimap())
```

Comment:
@DiogoMendonc-a you were picked up by the code owners stuff only because of this change. If you don't want to review the rest its okay :laughing: